plugins/languages/markdown/glow.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "glow";
+  originalName = "glow.nvim";
+  defaultPackage = pkgs.vimPlugins.glow-nvim;
+
+  maintainers = [ lib.maintainers.getchoo ];
+
+  settingsOptions = {
+    glow_path = helpers.defaultNullOpts.mkStr (helpers.mkRaw "vim.fn.exepath('glow')") ''
+      Path to `glow` binary
+
+      Consider using `glowPackage` instead
+    '';
+
+    install_path = helpers.defaultNullOpts.mkStr "~/.local/bin" ''
+      Path for installing `glow` binary if one is not found
+
+      Consider using `glowPackage` instead
+    '';
+
+    border = helpers.defaultNullOpts.mkEnumFirstDefault [
+      "shadow"
+      "none"
+      "double"
+      "rounded"
+      "solid"
+      "single"
+    ] "Style of the floating window's border";
+
+    style = helpers.defaultNullOpts.mkEnum [
+      "dark"
+      "light"
+    ] (lib.literalExpression "vim.opt.background") "Glow style";
+
+    pager = helpers.defaultNullOpts.mkBool false ''
+      Display output in a pager style
+    '';
+
+    width = helpers.defaultNullOpts.mkInt 100 ''
+      Width of the floating window
+    '';
+
+    height = helpers.defaultNullOpts.mkInt 100 ''
+      Height of the floating window
+    '';
+
+    width_ratio = helpers.defaultNullOpts.mkNum 0.7 ''
+      Maximum width of the floating window relative to the window size
+    '';
+
+    height_ratio = helpers.defaultNullOpts.mkNum 0.7 ''
+      Maximum height of the floating window relative to the window size
+    '';
+  };
+
+  settingsExample = {
+    border = "shadow";
+    style = "dark";
+    pager = false;
+    width = 80;
+    height = 100;
+    width_ratio = 0.7;
+    height_ratio = 0.7;
+  };
+
+  extraOptions = {
+    glowPackage = helpers.mkPackageOption {
+      name = "glow";
+      default = pkgs.glow;
+      defaultText = lib.literalExpression "pkgs.glow";
+    };
+  };
+
+  extraConfig = cfg: {
+    plugins.glow.settings = {
+      glow_path = lib.mkIf (cfg.glowPackage != null) (lib.getExe cfg.glowPackage);
+    };
```

Comment:
Or we can simply `extraPackages = [cfg.glowPackage]`. The plugin will be able to find it just fine.

Reply:
If so, we could reword the `glow_path` desc to something like:

"If left set to null, `config.glow.glowPackage` will be used via PATH"

(Could be worded better)